webdriver-spec.html

```diff
@@ -5395,6 +5395,23 @@
       to describe single actions, notably <a>pointerMove</a>, that
       occur in stages across a defined wall time interval.
 
+    <p>At the lowest level, the behaviour of actions is intended to
+    mimic the remote end's behaviour with an actual input device as
+    closely as possible, and the implementation strategy may involve
+    e.g. injecting fake input events into a browser event
+    loop. Therefore the steps to dispatch an action will inevitably
```

Comment:
I think it’s worth implying what “fake input event” means in this context, i.e. by referring to the word “synthesised”.